examples/common/scripts/vtctld-up.sh

```diff
@@ -36,6 +36,8 @@ vtctld \
  --pprof-http \
   > $VTDATAROOT/tmp/vtctld.out 2>&1 &
 
+echo "Curling \"http://${hostname}:${vtctld_web_port}/debug/status\" to check if vtctld is up"
```

Comment:
is it not the same thing we do from line 41 to 47?

Reply:
this just adding an echo so that we can see how far the script got, and where it might be stuck, if at all.

Reply:
Yes Deepthi is correct - just a sanity check that vtctld client was initiated, and we're moving onto curling now! 😄